sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_shared/_polling.py

```diff
@@ -75,6 +76,10 @@ def wait(self, timeout=None):
         except TypeError:  # Was None
             pass
 
+    @classmethod
+    def from_continuation_token(cls, polling_method, **kwargs):
```

Comment:
Strange that `from_continuation_token` takes a polling method as its only argument :suspect:

Reply:
I see your point. The reason I did it this way, is because creating the polling method is the same for with continuation token and without. Since we already have this code in the client, i didn't want to just pass in the continuation token and duplicate that code in the `from_continuation_token` method. I'm down to change this if you think it's too weird to pass in a polling method to `from_continuation_token`